internal/inventory/awsfetcher/fetcher_networking.go

```diff
@@ -96,135 +94,16 @@ func (s *networkingFetcher) fetch(ctx context.Context, resourceName string, func
 	for _, item := range awsResources {
 		assetChannel <- inventory.NewAssetEvent(
 			classification,
-			[]string{item.GetResourceArn(), pointers.Deref(s.retrieveId(item))},
+			item.GetResourceArn(),
 			item.GetResourceName(),
 			inventory.WithRawAsset(item),
-			inventory.WithCloud(inventory.AssetCloud{
-				Provider: inventory.AwsCloudProvider,
-				Region:   item.GetRegion(),
-				Account: inventory.AssetCloudAccount{
-					Id:   s.AccountId,
-					Name: s.AccountName,
-				},
-				Service: &inventory.AssetCloudService{
-					Name: "AWS Networking",
-				},
+			inventory.WithCloud(inventory.Cloud{
+				Provider:    inventory.AwsCloudProvider,
+				Region:      item.GetRegion(),
+				AccountID:   s.AccountId,
+				AccountName: s.AccountName,
+				ServiceName: "AWS Networking",
 			}),
-			s.networkEnricher(item),
```

Comment:
just to clarify - we're not sending non ECS compliant data anymore and that's why this enricher is removed?

Reply:
Yes, there was not no need for the data we were fetching